Description: Link with libGLU on Linux
 Debian Linux build also requires linking with libGLU.
 .
 tartini (1.2-1) unstable; urgency=low
 .
   * Initial release (Closes: #515003)
Author: Jeremy Salwen <jeremysalwen@gmail.com>
Bug-Debian: http://bugs.debian.org/515003

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- tartini-1.2.orig/pitch.pro
+++ tartini-1.2/pitch.pro
@@ -325,7 +325,7 @@ unix{
 
     DEFINES += LINUX
     INCLUDEPATH += $$MY_INCLUDE_PATH
-    LIBS += $$MY_LIB_PATH -lfftw3f -lqwt-qt4 -lasound
+    LIBS += $$MY_LIB_PATH -lGLU -lfftw3f -lqwt-qt4 -lasound
     CONFIG += warn_off
     QMAKE_CXXFLAGS += -Wall -Wno-non-virtual-dtor -pipe
     profile {
